wiki/Gameplay/Note_lock/en.md

```diff
@@ -0,0 +1,32 @@
+---
+tags:
+  - notelock
+  - jittering notes
+  - shaking notes
+---
+
+# Note lock
+
+![](img/notelock.gif "Example of note lock: the second note is inaccessible")
+
+**Note lock** is a gameplay mechanic of [osu!](/wiki/Game_mode/osu!) which may prevents a player from clearing a hit object. It happens if **two** conditions are met at the same time:
+
+1. The [timing windows](/wiki/Beatmapping/Overall_difficulty#timing) of two hit objects overlap.
+2. The first object of those two was either missed, or ignored by the player, and is still present on the play field.
```

Comment:
missed objects don't block future objects. only ignored.

Reply:
pretty sure the "miss" here refers to clicking outside the hit object (and moving on to the next one), while i think you mean "miss" as in the hit window running out (which happens later). something like this should be correct:

```suggestion
2. The first object of those two was not clicked and is still present on the play field.
```

Reply:
An alternative way of saying all this would be to combine both 1 and  2 into a single statement. Something like "if there's a previous object which has not been hit or missed (ie. not judged)"

Reply:
fixed, should be better now